app/routes/meetings/components/MeetingDetail.js

```diff
@@ -192,6 +192,9 @@ class MeetingDetails extends Component<Props> {
 
           <ContentSection>
             <ContentMain>
+              {meeting.description && (
+                <DisplayContent content={meeting.description} />
```

Comment:
I don't think we should use `DisplayContent` for non-rich-text. I'm not sure if we have some component to render plain text, but as discussed(https://github.com/webkom/lego/pull/2035#discussion_r564351182) we should maybe create it.

Reply:
Ahh, sure. Guess we can just use it as plain text in a `<p />` or something